util/log/clog.go

```diff
@@ -510,7 +510,7 @@ func formatHeader(s Severity, now time.Time, threadID int32, file string, line i
 	n += buf.nDigits(6, n, now.Nanosecond()/1000, '0')
 	tmp[n] = ' '
 	n++
-	n += buf.nDigits(7, n, int(threadID), ' ')
+	n += buf.someDigits(n, int(threadID))
```

Comment:
FYI, https://github.com/petermattis/goid works on go 1.4 and earlier but looks like it will break on go 1.5 (`getg` is now inlined by the compiler).